### ISSUE TITLE
Keyboard Navigation use with Simulation Tools?

### ISSUE THREAD

**Issue by DevonQui** [phetsims/wave-interference#511]
**Test device**
MacBook Pro

**Operating System**
Big Sur 11.0.1

**Browser**
Chrome

**Problem description**
This is for https://github.com/phetsims/QA/issues/580

I'm not entirely sure if this was intentional or not but when I was testing the keyboard navigation in the simulation, I was unable to access the tools above the wave settings. Tools such as the timer, measuring tape, etc. I haven't tested it in Safari yet but I'll follow up shortly.

**Steps to reproduce**
1. Open the simulation
2. Press 'Tab' to allow for keyboard navigation
3. Cycle through all the options by continually clicking 'Tab' and notice that you're never able to access the tool bar

**Visuals**
<img width="349" alt="Screen Shot 2021-01-04 at 12 12 07 PM" src="https://user-images.githubusercontent.com/72042459/103570302-1764d380-4e86-11eb-87cd-b8d24ce57698.png">

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Waves Intro‬
URL: https://phet-dev.colorado.edu/html/waves-intro/1.1.0-rc.3/phet/waves-intro_all_phet.html
Version: 1.1.0-rc.3 2020-12-08 22:49:34 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 11_1_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
Language: en-US
Window: 1440x821
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

**Comment by DevonQui** [comment]
I just tested this out in Safari as well and I was still unable to access the simulation tools using keyboard navigation.

**Comment by samreid** [comment]
The tools in the top toolbox are not designed to be keyboard accessible for this version.

**Comment by arouinfar** [comment]
Previous versions of the sim never advertised support for alt input, but we're officially adding support in the upcoming 2.1 release. The measurement tools now all work with alt input, closing.
